Data/src/main/java/org/tribuo/data/columnar/processors/response/QuartileResponseProcessor.java

```diff
@@ -60,9 +94,22 @@ private QuartileResponseProcessor() {}
      * @param outputFactory The output factory to use.
      */
     public QuartileResponseProcessor(String name, String fieldName, Quartile quartile, OutputFactory<T> outputFactory) {
+        this(Collections.singletonList(fieldName), Collections.singletonList(quartile), outputFactory);
         this.name = name;
-        this.fieldName = fieldName;
-        this.quartile = quartile;
+    }
+
+    /**
+     * Constructs a response processor which emits 4 distinct bins for the output factory to process.
+     * <p>
+     * This works best with classification outputs as the discrete binning is tricky to do in other output
+     * types.
+     * @param fieldNames The field to read.
+     * @param quartiles The quartile range to use.
+     * @param outputFactory The output factory to use.
+     */
+    public QuartileResponseProcessor(List<String> fieldNames, List<Quartile> quartiles, OutputFactory<T> outputFactory) {
+        this.fieldNames = fieldNames;
```

Comment:
These constructors are a bit weird now. Name used to be mandatory, and it's not clear why it's not anymore.